table/block_test.cc

```diff
@@ -68,6 +68,28 @@ void GenerateRandomKVs(std::vector<std::string> *keys,
   }
 }
 
+void GenerateRandomKVs(std::vector<std::string> *keys,
```

Comment:
V is actually a block handle. Can you rename the function so that it is clearer?